src/runner/QCheck_base_runner.ml

```diff
@@ -60,7 +60,14 @@ let set_seed_ ~colors s =
 (* time of last printed message. Useful for rate limiting in verbose mode *)
 let last_msg = ref 0.
 
-let time_between_msg = ref 0.1
+let time_between_msg =
+  let env_var = "QCHECK_MSG_INTERVAL" in
+  let default_interval = 0.1 in
+  let interval = match Sys.getenv_opt env_var with
+    | None -> default_interval
+    | Some f -> float_of_string f in
```

Comment:
What about replacing `float_of_string f` with:
```
match float_of_string_opt f with
| None -> invalid_arg (env_var ^ " must be a float")
| Some i -> i
```
?

Reply:
Fair enough, that should give a more descriptive error message to the user. Fixed in efd8f3e